services/notifications/pkg/service/service.go

```diff
@@ -34,12 +39,14 @@ func NewEventsNotifier(
 	logger log.Logger,
 	gwClient gateway.GatewayAPIClient,
 	machineAuthAPIKey, emailTemplatePath, ocisURL string) Service {
+
 	return eventsNotifier{
 		logger:            logger,
 		channel:           channel,
 		events:            events,
 		signals:           make(chan os.Signal, 1),
 		gwClient:          gwClient,
+		valueService:      settingssvc.NewValueService("com.owncloud.api.settings", grpc.DefaultClient()),
```

Comment:
Should be passed to the function. Not created here. Do it same way as `gwClient`

Reply:
fixed